src/NetworkConfiguration.py

```diff
@@ -8,7 +8,7 @@
     'MAINNET': {'NAME': 'MAINNET', 'NB_FREEZE_CYCLE': 5, 'BLOCK_TIME_IN_SEC': 60, 'BLOCKS_PER_CYCLE': 4096,
                 'BLOCKS_PER_ROLL_SNAPSHOT': 256},
     'ALPHANET': {'NAME': 'ALPHANET', 'NB_FREEZE_CYCLE': 3, 'BLOCK_TIME_IN_SEC': 30, 'BLOCKS_PER_CYCLE': 2048,
-                 'BLOCKS_PER_ROLL_SNAPSHOT': 256},
+                 'BLOCKS_PER_ROLL_SNAPSHOT': 128},
```

Comment:
This change means we have a bug in the current release?

Reply:
Yes, unfortunately. https://delphinet-tezos.giganode.io/chains/main/blocks/head/context/constants

Reply:
Please rename accoring to the conventions used in the Tezos repository. We maintain the networks: carthagenet, delphinet, etc and depricate them accorind to the shut down plans. e.g. carthage will be shut down 12.12.2020.